src/engine/strat_engine/mdv.rs

```diff
@@ -131,6 +136,21 @@ impl MetadataVol {
             }
         }
 
+        let fsinfo = try!(Statvfs::for_path(&self.mount_pt));
```

Comment:
These ```check``` functions are causing our code to deteriorate and are becoming a maintenance burden. It is not possible to make any sort of useful review of the code that we are adding to them at this point. Our best course is to rip out the check functions, saving the useful stuff that has already been added, and start again in a better organized fashion.